.github/workflows/po-linter.yaml

```diff
@@ -0,0 +1,54 @@
+name: Prometheus Operator Linter
+
+on:
+  pull_request:
+    paths:
+    - namespaces/live.cloud-platform.service.justice.gov.uk/*/*prometheus.yaml
+    - namespaces/live.cloud-platform.service.justice.gov.uk/*/*prometheus.yml
+    - namespaces/live-2.cloud-platform.service.justice.gov.uk/*/*/*prometheus.yaml
+    - namespaces/live-2.cloud-platform.service.justice.gov.uk/*/*prometheus.yml
+  workflow_dispatch:
+
+jobs:
+  po-lint:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run po-linter
+        uses: docker://ministryofjustice/cloud-platform-po-linter:latest
+        continue-on-error: true
+        id: po-linter
+        env:
+          GITHUB_OAUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          
+      - name: Result
+        uses: actions/github-script@v6
+        env:
+            summary: "Result:\n${{ steps.po-linter.outputs.po-linter }}"
+        with:
+            github-token: ${{ secrets.ACTION_TOKEN }}
```

Comment:
`ACTION_TOKEN` doesnot exists. Should it have different permissions than the default GITHUB_TOKEN?